lib/qx/tool/compiler/app/loader-server.tmpl.js

```diff
@@ -155,15 +155,9 @@
       },
 
       signalStartup : function() {
-        qx.Bootstrap.executePendingDefers();
-        qx.$$loader.delayDefer = false;
```

Comment:
I don't think that these can be removed, what's your rationale?

Reply:
You are right - this solves the problem that qx.bom.client.Browser is'nt loaded. Thanks for the hint